feat(ChatbotModal): Add style-able modal for general use (#307)
patternfly/virtual-assistant: packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/Chatbot/Chatbot.md

```js file="./ChatbotWelcomePrompt.tsx"

```

### Modal

> We offer a customized [PatternFly modal](/components/modal) for use in chatbots. It will adapt to the display mode and will accept components usually used in a modal. It is used and primarily tested in the context of the [attachment modals](/patternfly-ai/chatbot/chatbot-attachments#attachment-preview). It is being exposed so that consumers can also use it, but it may require customization for your specific use case.

Reviewer: ```suggestion
Based on the [PatternFly modal](/components/modal), this modal adapts to the chatbot display mode and accepts components typically used in a modal. It is primarily used and tested in the context of the [attachment modals](/patternfly-ai/chatbot/chatbot-attachments#attachment-preview), but you can customize this modal to adapt it to other use cases as needed.
```

How accurate do my changes seem? Would it be true to say "it can be customized using PatternFly modal properties to adapt to other use cases as needed."

Author: I think what you have in your suggestion is accurate. They can technically pass any Modal props or components to it, but I haven't rigorously tested it against everything we offer here: https://www.patternfly.org/components/modal. I can do more rigorous testing here before we merge this.
